kivy/data/style.kv

```diff
@@ -508,6 +508,7 @@
 # Popup widget
 <Popup>:
     _container: container
+    pos_hint: { "center_x": 0.5, "center_y": 0.5 } 
```

Comment:
`ModalView`, from which `Popup` inherits, has a method `_align_center` that is supposed to do this. Can you check why that is not working? Setting `pos_hint` here is not a good idea because it breaks backwards compat and users cannot control the value once set here.